Cannot find autostart amx option in Glassfish 7

## Environment Details
* GlassFish Version (and build number): 7.0.10
* JDK version: jdk-17.0.8.1_1-openjdk-adoptium
* OS: Windows 10
* Database: Postgresql

----------

## Problem Description

Coming from Payara 5 and 6 I can enable amx automatically on startup using
`<amx-configuration enabled="true"></amx-configuration>`
in domain.xml

However in Glassfish 7 I cannot seem to find such an option.
Via jconsole I can see the amx-support MBean and I can select the bootAMX() operation to start amx.

In the administration guide:
https://glassfish.org/docs/7.0.10/administration-guide.html#using-rest-interfaces-to-administer-glassfish-server
I can see a minimal mention of amx.

The goal I would like to achieve:
- from within an application / ear file I would like to ask some server state / settings from an M-Bean or REST call
- example: ask `amx:pp=/domain/configs/config[server-config],type=transaction-service` for attribute `TimeoutInSeconds`

## Steps to reproduce
Deploy ear file, call java.lang.management.ManagementFactory.getPlatformMBeanServer(); 
Perform query to "amx" object.
Object `amx:pp=/domain/configs/config[server-config],type=transaction-service` not found due to amx not being started.

Workaround start amx from the ear file using MBeanServer like:
```
final MBeanServer platformMBeanServer = ManagementFactory.getPlatformMBeanServer();
try {
	ObjectName objectName = new ObjectName("amx-support:type=boot-amx");
	if (platformMBeanServer.isRegistered(objectName)) {
		platformMBeanServer.invoke(objectName, "bootAMX", new Object[0], new String[0]);
	} else {
		log.warn("amx-support:type=boot-amx not available");
	}
} catch (Exception e) {
	log.error("Unable to start AMX", e);
}
```

## Impact of Issue
Main question really is:
- can amx be started automatically like in Payara via domain.xml? if so: then the bug would be: some documentation is missing about the existence of the bootAmx operation for this purpose
- is amx still valid to use? or is it deprecated and should the management REST Api be used instead?
-- But it adds quite some overhead to get some server information from within the ear file.

## Logging issue
When starting AMX in a clean domain you get errors / exceptions in INFO messages:
**IllegalArgumentException: ResourceRef refers to non-existent resource: GlassFishConfigBean.com.sun.enterprise.config.serverbeans.ResourceRef**
```
[#|2024-01-24T13:26:57.296964+01:00|INFO|GF 7.0.10|jakarta.enterprise.system.tools.amx|_ThreadID=244;_ThreadName=Thread-34;_LevelValue=800;_MessageID=NCLS-COM-00008;|
  AMX Startup Service: AMXLoader failed to load {0}
javax.management.RuntimeMBeanException: RuntimeException thrown in postRegister method: rethrowing <java.lang.RuntimeException: javax.management.RuntimeMBeanException: RuntimeException thrown in postRegister method: rethrowing <java.lang.IllegalArgumentException: ResourceRef refers to non-existent resource: GlassFishConfigBean.com.sun.enterprise.config.serverbeans.ResourceRef>, but keeping the MBean registered>, but keeping the MBean registered
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.postRegister(DefaultMBeanServerInterceptor.java:1023)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerDynamicMBean(DefaultMBeanServerInterceptor.java:968)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:895)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:320)
        at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:523)
        at com.sun.enterprise.v3.admin.DynamicInterceptor.registerMBean(DynamicInterceptor.java:396)
        at org.glassfish.admin.amx.impl.j2ee.loader.AMXJ2EEStartupService.loadAMXMBeans(AMXJ2EEStartupService.java:216)
        at org.glassfish.admin.amx.impl.AMXStartupService$AMXLoaderThread.run(AMXStartupService.java:265)
Caused by: java.lang.RuntimeException: javax.management.RuntimeMBeanException: RuntimeException thrown in postRegister method: rethrowing <java.lang.IllegalArgumentException: ResourceRef refers to non-existent resource: GlassFishConfigBean.com.sun.enterprise.config.serverbeans.ResourceRef>, but keeping the MBean registered
        at org.glassfish.admin.amx.impl.mbean.AMXImplBase.registerChild(AMXImplBase.java:954)
        at org.glassfish.admin.amx.impl.j2ee.J2EEDomainImpl.registerChildren(J2EEDomainImpl.java:82)
        at org.glassfish.admin.amx.impl.mbean.AMXImplBase.postRegisterHook(AMXImplBase.java:898)
        at org.glassfish.admin.amx.impl.mbean.MBeanImplBase.postRegister(MBeanImplBase.java:327)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.postRegister(DefaultMBeanServerInterceptor.java:1018)
        ... 7 more
Caused by: javax.management.RuntimeMBeanException: RuntimeException thrown in postRegister method: rethrowing <java.lang.IllegalArgumentException: ResourceRef refers to non-existent resource: GlassFishConfigBean.com.sun.enterprise.config.serverbeans.ResourceRef>, but keeping the MBean registered
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.postRegister(DefaultMBeanServerInterceptor.java:1023)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerDynamicMBean(DefaultMBeanServerInterceptor.java:968)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:895)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:320)
        at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:523)
        at com.sun.enterprise.v3.admin.DynamicInterceptor.registerMBean(DynamicInterceptor.java:396)
        at org.glassfish.admin.amx.impl.mbean.AMXImplBase.registerChild(AMXImplBase.java:950)
        ... 11 more
Caused by: java.lang.IllegalArgumentException: ResourceRef refers to non-existent resource: GlassFishConfigBean.com.sun.enterprise.config.serverbeans.ResourceRef
        at org.glassfish.admin.amx.impl.j2ee.RegistrationSupport.processResourceRef(RegistrationSupport.java:554)
        at org.glassfish.admin.amx.impl.j2ee.RegistrationSupport$RefListener.startListening(RegistrationSupport.java:673)
        at org.glassfish.admin.amx.impl.j2ee.RegistrationSupport.start(RegistrationSupport.java:136)
        at org.glassfish.admin.amx.impl.j2ee.J2EEServerImpl.registerChildren(J2EEServerImpl.java:98)
        at org.glassfish.admin.amx.impl.j2ee.DASJ2EEServerImpl.registerChildren(DASJ2EEServerImpl.java:49)
        at org.glassfish.admin.amx.impl.mbean.AMXImplBase.postRegisterHook(AMXImplBase.java:898)
        at org.glassfish.admin.amx.impl.mbean.MBeanImplBase.postRegister(MBeanImplBase.java:327)
        at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.postRegister(DefaultMBeanServerInterceptor.java:1018)
        ... 17 more
|#]
[#|2024-01-24T14:21:04.131441+01:00|INFO|GF 7.0.10|jakarta.enterprise.system.tools.amx|_ThreadID=137;_ThreadName=AutoDeployer;_LevelValue=800;_MessageID=NCLS-COM-00009;|
  AMX Startup Service: AMX ready for use, DomainRoot amx:pp=,type=domain-root|#]
```
however AMX Startup completed and amx MBeans are available and can be used.

Workaround to hide these messages is to change logging.properties to change existing line:
`jakarta.enterprise.system.tools.amx.level=INFO `
to
`jakarta.enterprise.system.tools.amx.level=WARNING`
but this will hide the 
`AMX Startup Service: AMX ready for use`
message as well.

And I get some System.out log lines 8 times the same __defaultManagedThreadFactory already exists exception, which I would not expect:
```
[#|2024-01-24T14:45:28.562217+01:00|INFO|GF 7.0.10|jakarta.enterprise.logging.stdout|_ThreadID=224;_ThreadName=AMXConfigLoader.AMXConfigLoaderThread;_LevelValue=800;|
  javax.management.InstanceAlreadyExistsException: "amx:pp=/domain/servers/server[server],type=resource-ref,name=concurrent/__defaultManagedThreadFactory"
java.management/com.sun.jmx.mbeanserver.Repository.addMBean(Repository.java:436)
java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerWithRepository(DefaultMBeanServerInterceptor.java:1865)
java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerDynamicMBean(DefaultMBeanServerInterceptor.java:960)
java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerObject(DefaultMBeanServerInterceptor.java:895)
java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.registerMBean(DefaultMBeanServerInterceptor.java:320)
java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.registerMBean(JmxMBeanServer.java:523)
com.sun.enterprise.v3.admin.DynamicInterceptor.registerMBean(DynamicInterceptor.java:396)
org.glassfish.admin.amx.impl.config.AMXConfigLoader.createAndRegister(AMXConfigLoader.java:478)
org.glassfish.admin.amx.impl.config.AMXConfigLoader._registerConfigBeanAsMBean(AMXConfigLoader.java:456)
org.glassfish.admin.amx.impl.config.AMXConfigLoader.registerConfigBeanAsMBean(AMXConfigLoader.java:428)
org.glassfish.admin.amx.impl.config.AMXConfigLoader$AMXConfigLoaderThread.registerOne(AMXConfigLoader.java:357)
org.glassfish.admin.amx.impl.config.AMXConfigLoader$AMXConfigLoaderThread.doRun(AMXConfigLoader.java:398)
org.glassfish.admin.amx.impl.config.AMXConfigLoader$AMXConfigLoaderThread.run(AMXConfigLoader.java:371)|#]
```
Ideally these should be behind some log level.

--- a/appserver/common/amx-jakartaee/src/main/java/org/glassfish/admin/amx/impl/j2ee/loader/AMXJ2EEStartupService.java
+++ b/appserver/common/amx-jakartaee/src/main/java/org/glassfish/admin/amx/impl/j2ee/loader/AMXJ2EEStartupService.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2006, 2022 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -60,10 +61,6 @@ public final class AMXJ2EEStartupService
         implements org.glassfish.hk2.api.PostConstruct,
         org.glassfish.hk2.api.PreDestroy,
         AMXLoader, ConfigListener {
-
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
 
     @Inject
     private MBeanServer mMBeanServer;

--- a/nucleus/admin/config-api/src/main/java/org/glassfish/config/support/GlassFishConfigBean.java
+++ b/nucleus/admin/config-api/src/main/java/org/glassfish/config/support/GlassFishConfigBean.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -17,6 +18,8 @@
 package org.glassfish.config.support;
 
 import java.lang.reflect.Proxy;
+import java.util.StringJoiner;
+
 import javax.xml.stream.XMLStreamReader;
 
 import org.glassfish.hk2.api.ServiceLocator;
@@ -96,7 +99,10 @@ public final class GlassFishConfigBean extends ConfigBean {
     }
 
     public String toString() {
-        //final Set<String> attrNames = getAttributeNames();
-        return "GlassFishConfigBean." + getProxyType().getName();
+        StringJoiner attributeJoiner = new StringJoiner(",");
+        for (String attributeName : getAttributeNames()) {
+            attributeJoiner.add(attributeName + "=" + attribute(attributeName));
+        }
+        return "GlassFishConfigBean." + getProxyType().getName() + "(" + attributeJoiner.toString() + ")";
     }
 }

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/base/MBeanTracker.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/base/MBeanTracker.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -38,9 +39,6 @@ import org.glassfish.external.arc.Taxonomy;
 @AMXMBeanMetadata(singleton = true, globalSingleton = true, leaf = true)
 public final class MBeanTracker implements NotificationListener, MBeanRegistration, MBeanTrackerMBean {
 
-    private static void debug(final Object o) {
-        System.out.println("" + o);
-    }
     /**
      * maps a parent ObjectName to a Set of children
      */

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/core/PathnameParser.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/core/PathnameParser.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -14,10 +15,6 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  */
 
-/*
- * To change this template, choose Tools | Templates
- * and open the template in the editor.
- */
 package org.glassfish.admin.amx.core;
 
 import java.util.ArrayList;
@@ -40,11 +37,6 @@ import org.glassfish.external.arc.Taxonomy;
 @Taxonomy(stability = Stability.UNCOMMITTED)
 public final class PathnameParser
 {
-    private static void debug(final Object o)
-    {
-        System.out.println("" + o);
-    }
-
     private final char mDelim;
 
     private final char mNameLeft;

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/core/Util.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/core/Util.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -50,10 +51,6 @@ import static org.glassfish.external.amx.AMX.TYPE_KEY;
 public final class Util {
 
     private static final String QUOTE_CHAR = "\"";
-
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
 
     public static String quoteIfNeeded(String name) {
         if(name.indexOf(":") > 1) {

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/AMXStartupService.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/AMXStartupService.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022, 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2006, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -65,10 +65,6 @@ public final class AMXStartupService
         implements org.glassfish.hk2.api.PostConstruct,
         org.glassfish.hk2.api.PreDestroy,
         AMXStartupServiceMBean {
-
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
 
     @Inject
     ServiceLocator mHabitat;

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AMXConfigLoader.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AMXConfigLoader.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2008, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -53,9 +54,6 @@ Responsible for loading ConfigBeanProxy MBeans (com.sun.enterprise.config.server
 public final class AMXConfigLoader
         implements TransactionListener {
 
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
     private volatile AMXConfigLoaderThread mLoaderThread;
     private final Transactions mTransactions;
     private final Logger mLogger = AMXLoggerInfo.getLogger();
@@ -209,7 +207,7 @@ public final class AMXConfigLoader
                         issueAttributeChange(cb, propertyName, oldValue, newValue, whenChanged);
                     }
                 } else {
-                    debug("AMXConfigLoader.sortAndDispatch: WARNING: source is not a ConfigBean");
+                    mLogger.fine("AMXConfigLoader.sortAndDispatch: WARNING: source is not a ConfigBean");
                 }
             }
         }
@@ -482,8 +480,7 @@ public final class AMXConfigLoader
             //System.out.println( "AMXConfigLoader.createAndRegister(): REGISTERED: " + objectName + " at " + System.currentTimeMillis() );
             //System.out.println( JMXUtil.toString( mServer.getMBeanInfo(objectName) ) );
         } catch (final JMException e) {
-            debug(ExceptionUtil.toString(e));
-
+            mLogger.log(Level.WARNING, AMXLoggerInfo.cantRegister, new Object[]{getType(cb), getKey(cb), e});
             objectName = null;
         }
         return objectName;

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AMXConfigStartupService.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AMXConfigStartupService.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2006, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -55,9 +56,6 @@ public final class AMXConfigStartupService
         org.glassfish.hk2.api.PreDestroy,
         AMXLoader {
 
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
     @Inject
     InjectedValues mInjectedValues;
     @Inject//(name=AppserverMBeanServerFactory.OFFICIAL_MBEANSERVER)

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AttributeResolverHelper.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/AttributeResolverHelper.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2011, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -42,10 +43,6 @@ import org.jvnet.hk2.config.VariableResolver;
  */
 @Taxonomy( stability = Stability.UNCOMMITTED)
 public class AttributeResolverHelper extends VariableResolver {
-
-    private static void debug(final String s) {
-        System.out.println("##### " + s);
-    }
 
     public AttributeResolverHelper(final AMXConfigProxy amx) {
     }

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/ConfigBeanJMXSupport.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/ConfigBeanJMXSupport.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2023, 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -1601,12 +1601,6 @@ class ConfigBeanJMXSupport
         }
         return m;
     }
-
-    private static void debug(final String s)
-    {
-        System.out.println("### " + s);
-    }
-
 }
 
 

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/ConfigBeanRegistry.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/ConfigBeanRegistry.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -13,12 +14,6 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  */
-
-/*
- * To change this template, choose Tools | Templates
- * and open the template in the editor.
- */
-
 package org.glassfish.admin.amx.impl.config;
 
 import java.util.concurrent.ConcurrentHashMap;
@@ -35,7 +30,6 @@ import org.jvnet.hk2.config.Dom;
  */
 @Taxonomy( stability=Stability.NOT_AN_INTERFACE )
 public final class ConfigBeanRegistry {
-    private static void debug( final String s ) { System.out.println(s); }
 
     public static final class MBeanInstance
     {

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/mbean/ComplianceMonitor.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/mbean/ComplianceMonitor.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -213,10 +214,6 @@ public final class ComplianceMonitor implements NotificationListener {
                 }
             }
         }
-    }
-
-    private static void debug(final Object o) {
-        System.out.println(o.toString());
     }
 }
 

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/ImplUtil.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/ImplUtil.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -23,10 +24,6 @@ import org.glassfish.admin.amx.core.AMXProxy;
 import org.glassfish.admin.amx.core.proxy.ProxyFactory;
 
 public final class ImplUtil {
-
-    private static void debug(final String s) {
-        System.out.println(s);
-    }
 
     /**
     Unload this AMX MBean and all its children.

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/MBeanInfoSupport.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/MBeanInfoSupport.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -12,11 +13,6 @@
  * https://www.gnu.org/software/classpath/license.html.
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-/*
- * To change this template, choose Tools | Templates
- * and open the template in the editor.
  */
 package org.glassfish.admin.amx.impl.util;
 
@@ -61,9 +57,6 @@ public final class MBeanInfoSupport {
     private MBeanInfoSupport() {
     }
 
-    private static void debug(final Object o) {
-        System.out.println(o.toString());
-    }
     private static MBeanInfo amxspiMBeanInfo = null;
 
     public static synchronized MBeanInfo getAMX_SPIMBeanInfo() {

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/ObjectNameBuilder.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/ObjectNameBuilder.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -53,11 +54,6 @@ public final class ObjectNameBuilder {
 
         mParent = parent;
         mJMXDomain = parent.getDomain();
-    }
-
-    private static void debug(final Object o) {
-        System.out.println("" + o);
-        //AMXDebug.getInstance().getOutput( "org.glassfish.admin.amx.support.ObjectNameBuilder" ).println( o );
     }
 
     public String getJMXDomain() {

--- a/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/UnregistrationListener.java
+++ b/nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/util/UnregistrationListener.java
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation.
  * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
@@ -48,10 +49,6 @@ public final class UnregistrationListener implements NotificationListener {
                 mLatch.countDown();
             }
         }
-    }
-
-    private static void cdebug(final String s) {
-        System.out.println(s);
     }
 
     /**

